PermissionsEx/permissions.yml

```diff
@@ -12418,7 +12418,7 @@ groups:
         - ch.alias.haste2
         - ch.alias.haste3
         - ch.alias.haste4
-  wolverine:
+   wolverine:
```

Comment:
you gave wolverine 3 spaces, he should also only have 2

Reply:
Oh ok I will fix that too